<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager_dashboard</class>
 <widget class="QWidget" name="manager_dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager Dashboard</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="welcomeLabel">
     <property name="text">
      <string>Welcome, Manager!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 22pt 'Segoe UI'; color: #1976d2; margin: 20px 0 10px 0;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="restaurantsTab">
      <attribute name="title">
       <string>Restaurants</string>
      </attribute>
      <layout class="QVBoxLayout" name="restaurantsTabLayout">
       <!-- QTableWidget will be added dynamically in code -->
      </layout>
     </widget>
     <widget class="QWidget" name="usersTab">
      <attribute name="title">
       <string>Users</string>
      </attribute>
      <layout class="QVBoxLayout" name="usersTabLayout">
       <!-- QTableWidget will be added dynamically in code -->
      </layout>
     </widget>
     <widget class="QWidget" name="authApplicationsTab">
      <attribute name="title">
       <string>Restaurant Auth Applications</string>
      </attribute>
      <layout class="QVBoxLayout" name="authApplicationsTabLayout">
       <!-- QTableWidget will be added dynamically in code -->
      </layout>
     </widget>
     <widget class="QWidget" name="ordersAnalysisTab">
      <attribute name="title">
       <string>Orders Analysis</string>
      </attribute>
      <layout class="QVBoxLayout" name="ordersAnalysisTabLayout">
       <!-- QTableWidget will be added dynamically in code -->
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 